tools/validators/instance_validator/README.md

```diff
@@ -36,9 +36,10 @@ Note: as of the current development stage, you must clone the entire repository
 
 The validator supports a telemetry validation mode. When this mode is enabled, the validator will listen on a provided pubsub subscription for telemetry messages, and validate the message contents against the instance configuration. It is recommended that you first use the instance validator with telemetry validation mode disabled, and then enable it after that passes.
 
-If you would like to use the telemetry validation mode, you must provide the `--subscription` parameter, and you must either:
```

Comment:
this is Nour's pull request, you are overriting it.

Reply:
Reverted README to Nours PR.